build.sbt

```diff
@@ -92,24 +92,13 @@ lazy val testStdLayout    = Project("daffodil-test-stdLayout", file("test-stdLay
 lazy val commonSettings = Seq(
   organization := "org.apache.daffodil",
   version := "2.2.0-SNAPSHOT",
-  scalaVersion := "2.11.8",
+  scalaVersion := "2.12.6",
   scalacOptions ++= Seq(
```

Comment:
I think we should be able to add ``crossScalaVersion := Seq("2.12.6", "2.11.12")`` to enable support for cross building with 2.11. I believe Scala 2.12 requires Java 8, and unfortunately we do have some users that are still stuck on Java 7, even though we officially only support Java 8, so ensuring that stil works would be good.

Reply:
Related, we should also modify .travis.yml to include Scala versions 2.12.6 and 2.11.12 so that tests are run on both versions.